setupkoenv.lua

```diff
@@ -3,7 +3,7 @@ package.path =
     "common/?.lua;rocks/share/lua/5.1/?.lua;frontend/?.lua;" ..
     package.path
 package.cpath =
-    "common/?.so;common/?.dll;/usr/lib/lua/?.so;rocks/lib/lua/5.1/?.so;" ..
+    "common/?.so;common/?.dll;/usr/lib/lua/?.so;rocks/lib/lua/5.1/?.so;rocks/lib64/lua/5.1/?.so;" ..
```

Comment:
I'm not entirely sure if that's the best way to proceed (e.g., see [some of this stuff](https://github.com/LuaJIT/LuaJIT/blob/f2b37f51f69ba7e9f8defd05e5aca648121f0fe6/doc/install.html#L699-L752)), but in any case it isn't clearly the wrong thing to do. ;-)

As another example, it could be shortened and possibly made more future-proof by doing something like:
```suggestion
    "common/?.so;common/?.dll;/usr/lib/lua/?.so;rocks/?/lua/5.1/?.so;" ..
```

----

@NiLuJe Any thoughts?

Reply:
maybe `lib?`?

Reply:
I forget otoh what kind of pattern matching is used in this context. Typically I associate ? with at least *something*, not nothing. But if `lib?/lua` matches `lib/lua`, go for it.

Reply:
`?` is replaced by the module name, so, nope, that wouldn't do, unless fedora is even crazier than it looks ;).